css/styles.css

```diff
@@ -0,0 +1,48 @@
+* {
+  font-family: sans-serif;
```

Comment:
We should be using 2 different fonts in this project. You'll need to import these from Google fonts. We did this in `css-fonts`.

1. Open Sans 
2. Proza Libre (for the headings)

Reply:
updated :)